### ISSUE TITLE
Check config loading problems

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#2266]
**Describe the bug**
Report that aceconfig.ts no longer loads. Might be because require is no longer implicitly adding .js. Also a thought about walking parent directories if the config is not found in the working directory (such as how package.json works).

See https://ibm-studios.slack.com/archives/C036P1CTN/p1746128251211649

## Definition of Done

- [ ] Work item implemented
- [ ] Automated test case created (when applicable)
- [ ] Unit test completed
- [ ] Run Checker (when applicable)
- [ ] PR created

**Comment by philljenkins** [comment]
@tombrunet Should this be fixed prior to dropping support for 3.x?

**Comment by tombrunet** [comment]
Will be included in next release
